docker-compose.yml

```diff
@@ -14,8 +14,10 @@ services:
       - ./backend/db-setup/reset-db.sql:/usr/local/lib/reset-db.sql
       - ./backend/db-setup/reset-metabase-db.sql:/usr/local/lib/reset-metabase-db.sql
     command: -p ${SR_DB_PORT:-5432}
-    expose: 
+    expose:
       - "${SR_DB_PORT:-5432}"
+    ports:
+      - "5432:5432"
```

Comment:
This doesn't take the `SR_DB_PORT` env var into account - meaning if the port is set to anything other than 5432, the db won't be mounted on localhost